05_iigo/index.tex

```diff
@@ -500,4 +500,226 @@ \subsection{Legislative Branch}
+In this example there are two separate conditions. We can write both conditions as separate rules.
+Condition 1:
+\[
+    \textrm{Rule matrix: }
+    \begin{bmatrix}
+        0 & 2 & -1 & 0
+    \end{bmatrix}
+    \textrm{   Variables to fill:  }
+    \begin{bmatrix}
+        \textrm{Amount of tax agent is asked to pay} \\
+        \textrm{Amount of tax the agent is actually paying} \\
+        \textrm{Amount of resources agent has} \\
+        1
+    \end{bmatrix}     
+\]
+\[
+    \textrm{   Auxiliary vector: } 
+    \begin{bmatrix}
+        2 
+    \end{bmatrix}   
+\]
+\linebreak
+Condition 2:
+\[
+    \textrm{Rule matrix: }
+    \begin{bmatrix}
+        -1 & 1 & 0 & 0
+    \end{bmatrix}
+    \textrm{   Variables to fill:  }
+    \begin{bmatrix}
+        \textrm{Amount of tax agent is asked to pay} \\
+        \textrm{Amount of tax the agent is actually paying} \\
+        \textrm{Amount of resources agent has} \\
+        1
+    \end{bmatrix}     
+\]
+\[
+    \textrm{   Auxiliary vector: } 
+    \begin{bmatrix}
+        2 
+    \end{bmatrix}   
+\]
+However, we can also write this as a single stacked rule:
+\[
+    \textrm{Rule matrix: }
+    \begin{bmatrix}
+        0 & 2 & -1 & 0 \\
+        -1 & 1 & 0 & 0 
+    \end{bmatrix}
+    \textrm{   Variables to fill:  }
+    \begin{bmatrix}
+        \textrm{Amount of tax agent is asked to pay} \\
+        \textrm{Amount of tax the agent is actually paying} \\
+        \textrm{Amount of resources agent has} \\
+        1
+    \end{bmatrix}     
+\]
+\[
+    \textrm{   Auxiliary vector: } 
+    \begin{bmatrix}
+        2 \\
+        2 
+    \end{bmatrix}   
+\]
+The real power of this representation of rules is that the mechanics of the calculation is exposed. 
+Any agent can inspect the elements of the rule matrix and simply calculate what values they must provide to pass the rule.
+For example, if an agent is looking at the above rule and knows that their resource count is 120, and also know they've been asked to pay 75, they can go line by line through the matrix to work our what the minimum amount of tax that is needed to be paid.
+\[
+    \textrm{Matrix line 1: }
+    \begin{bmatrix}
+        0 & 2 & -1 & 0
+    \end{bmatrix}
+\]
+\[
+    \textrm{Values to be fed in: }
+    \begin{bmatrix}
+        75 \\
+        x \\
+        120 \\
+        1
+    \end{bmatrix}
+\]
+\[
+    \textrm{Agent calculation: }
+    0*75 + 2x -120 >= 0
+\]
+\[
+    x >= 60
+\]
+A similar calculation for the next line of the matrix yields:
+\[
+    \textrm{Agent calculation: }
+    x >= 75
+\]
+The agent can understand now to pay a minimum of 75. This system allows Agents to both check if they are compliant, and if not they can calculate which values they need to be compliant.
+As a final note, we have designed an additional abstraction to this rules system to accomodate conditional rules.
+For example if you had the rule "If resources are above 100, you must pay at least 75". 
```

Comment:
plz plz do this
`\begin{rule} \end{rule}` thingy....
AND go to math mode for numbers
example:
`$100$`